website/server/models/task.js

```diff
@@ -71,6 +71,13 @@ export let TaskSchema = new Schema({
     taskId: {type: String, ref: 'Task', validate: [validator.isUUID, 'Invalid uuid.']},
   },
 
+  requiresApproval: {type: Boolean, default: false},
```

Comment:
@TheHollidayInn this is not working because you cannot pass single nested props but have to pass the entire object. I would suggest to pass `{text, type, groupRequiresApproval}` and then based on the value of `groupRequiresApproval` adapt group.requiresApproval in the task. Or you could use a query parameter